tools/spelling_dictionary.txt

```diff
@@ -236,6 +236,8 @@ SRDS
 SRV
 SS
 SSL
+STATRICIA
```

Comment:
nit: can we pick a different dictionary word to avoid adding this to the dictionary?

Reply:
Done.